### ISSUE TITLE
"No friends" even though multiple people sharing their locations

### ISSUE THREAD

**Issue by zachberger** [BlueBubblesApp/bluebubbles-app#2687]
When I open FindMy it accurately shows the location of my items but when I switch to the friend tab it does not show me any locations, even though multiple people are sharing their locations with me.

![Screenshot_20240224-204950](https://github.com/BlueBubblesApp/bluebubbles-app/assets/1634878/78df9c2e-c9b9-4ca9-89d7-fca08659cc34)

I have Private APIs setup and other features dependent on this are working. I can see the friends locations fine on the Mac hosting the server.

**Comment by jjoelj** [comment]
What MacOS version is your server on?

**Comment by zachberger** [comment]
14.3.1

**Comment by jjoelj** [comment]
> 14.3.1

Find My Friends is not supported on MacOS Sonoma yet.

**Comment by zachberger** [comment]
Thank you for the info. Should we leave this open as a feature request? I had searched the bugs here and hadn't seen anything else mentioning it

**Comment by jjoelj** [comment]
Yeah, we can do that. I'd re-title it to be more specific, though.

**Comment by zachberger** [comment]
Updated, thank you!

**Comment by jjoelj** [comment]
Just realized this issue is more suited for the [BlueBubbles Helper](https://github.com/BlueBubblesApp/bluebubbles-helper) repo. I will transfer it there
